operator/config/crd/bases/cluster.redpanda.com_redpandas.yaml

```diff
@@ -9790,6 +9790,20 @@ spec:
                   reconcile request value, so a change of the annotation value
                   can be detected.
                 type: string
+              license:
+                description: |-
+                  LicenseStatus contains information about the current state of any
+                  installed license in the Redpanda cluster.
+                properties:
+                  expiration:
+                    type: string
+                  expired:
```

Comment:
It might be useful to report everything you get in `rpk`, which includes the following extra data points:

- License violation: A boolean value that indicates whether enterprise features are being used in violation of the licensing terms:

    - `true`: Enterprise features are enabled without a valid license.

    - `false`: No violations are detected.

- Organization: The organization for which the license was issued. Typically corresponds to the organization that purchased or activated the license.

- Type: The type of license applied to the cluster:

    - `enterprise`: A full-featured enterprise license.
    - `trial`: A time-limited trial license.

Reply:
I'll let @david-yu opine on the above - which we find useful, I believe we should have access to pretty much all of the above via the two API calls we're doing already, it'd just be a matter of plumbing it through to the status.